README.md

```diff
@@ -9,6 +9,12 @@ This the source for the Racket packages: "string-constants",
 Contribute to Racket by submitting a [pull request] or reporting an
 [issue]. See also the "Community" section of the [racket web page].
 
+You can generate the documentation for the package and get a short
+guide on translating by running:
+
+    scribble string-constants-doc/string-constants/string-constants.html
+
+
```

Comment:
Probably we want to include this guide in the documentation proper and then it will be available at docs.racket-lang.org, so we should point people there from the readme instead of requiring them to install racket.

Reply:
I will use https://docs.racket-lang.org/string-constants - the guide will probably be linked from there